docs/java-configuration.md

```diff
@@ -4,6 +4,91 @@ title: Scope Java Agent configuration
 sidebar_label: Configuration
 ---
 
+The behaviour of the Scope Agent can be modified by using  environment variables or by using a configuration file `Scope.yml`in the source root of the project.
+
+> If these properties are manually configured, they will be `true` only on encountering the string `true` configured on the environment variable. Any other value will be considered as `false`.
+
+## Disabling Scope instrumentation
+
+You can disable Scope auto instrumentation. Scope auto instruments your code and tests by default, you can disable this behaviour.
+
+### Using Environment Variables
+
+```sh
+SCOPE_AUTO_INSTRUMENT=false
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  auto_instrument: false
```

Comment:
In Java is:
```yaml
scope:
  instrumentation:
    enabled: true
```

Reply:
as in dotnet, so the env-var is: `SCOPE_INSTRUMENTATION_ENABLED `

Reply:
changed